REVIEWING.md

```diff
@@ -0,0 +1,73 @@
+
+### Trust
+
+Be trustworthy. During a review, your actions both build and help maintain the trust that the community has placed in this project. Below are examples of ways that we build trust:
+
+* **Transparency** - If a pull request won't be merged, clearly say why and close it. If a pull request won't be reviewed for a while, let the author know so they can set expectations and understand why it's blocked.
+* **Integrity** - Put the project's best interests ahead of personal relationships or company affiliations when deciding if a change should be merged.
+* **Stability** - Only merge when then change won't negatively impact project stability. It can be tempting to merge a pull request that doesn't meet our quality standards, for example when the review has been delayed, or because we are trying to deliver new features quickly, but regressions can significantly hurt trust in our project.
+
+## Process
+
+### PR Lifecycle
+
+#### 1. PR creation
+
+* Ensure that an issue has been created to track the feature enhancement or bug that is being fixed.
+* In the PR description, make sure you've included "Fixes #{issue_number}" e.g. "Fixes #242" so that GitHub knows to link it to an issue.
+* To avoid multiple contributors working on the same issue, please add a comment to the issue to let us know you plan to work on it.
+* If a significant amount of design is required, please include a proposal in the issue and wait for approval before working on code. If there's anything you're not sure about, please feel free to discuss this in the issue. We'd much rather all be on the same page at the start, so that there's less chance that drastic changes will be needed when your pull request is reviewed.
+
+#### 2. Reviewing/Discussion
+
+* Everyone is encouraged to participate in code reviews.
+* It is best to get approval from contributors with knowledge about the code paths being modified. You can identify an appropriate reviewer by looking through the file history to find frequent contributor. Contributors can not be directly added as reviews but can be notified via a comment.
+* When you provide feedback, make it clear if the change must be made for the pull request to be approved, or if it is just a suggestion. Mark suggestions with `nit`.
+* If your code review has had no activity for 7 days, a maintainer should assist in reassigning the review.  If a maintainer has not updated your review, please reach out to @project maintainers.
+
+#### 3. Merge or close
+
+* Maintainers are responsible for handling final approval and merging of pull requests. Contributors can approve pull requests and provide input.
+* Maintainers can merge their own pull requests after being approved by other maintainers.
+
+## Checklist
+
+Below are a set of common questions that apply to all pull requests:
```

Comment:
Are we able to create a pull request template that includes these items?

Reply:
Great suggestion! Looks like a template.md file can be added, created an [issue](https://github.com/deislabs/ratify/issues/175) to track this work.